package.json

```diff
@@ -11,14 +11,16 @@
   "scripts": {
     "build": "run-p --print-label build:lyra-api build:docs build:extension",
     "build:netlify": "run-s build:lyra-api build:docs",
-    "build:extension-preview": "run-s build:lyra-api build:extension",
+    "build:extension-preview": "run-s build:lyra-api build:extension:experimental",
```

Comment:
let's *try* having all previews build with the experimental flag. We will likely find it necessary as we start reviewing new feature PR's